python/ray/tune/result_grid.py

```diff
@@ -91,6 +93,15 @@ def experiment_path(self) -> str:
         location (path on the head node)."""
         return self._experiment_analysis.experiment_path
 
+    @property
+    def filesystem(self) -> pyarrow.fs.FileSystem:
```

Comment:
keep this named `storage_filesystem`? fine either way.

Reply:
I think it's best to be consistent with `checkpoint`, which uses `filesystem` as the attribute.